src/adapters/Wormhole/WormholeSenderAdapter.sol

```diff
@@ -35,8 +35,8 @@ contract WormholeSenderAdapter is IBridgeSenderAdapter, BaseSenderAdapter {
     }
 
     modifier onlyCaller() {
-        if (!gac.isPrevilagedCaller(msg.sender)) {
-            revert Error.INVALID_PREVILAGED_CALLER();
+        if (!gac.isprivilagedCaller(msg.sender)) {
+            revert Error.INVALID_PRIVILAGED_CALLER();
         }
         _;
```

Comment:
given this seems to be common logic across adapters should it be included in the `BaseSenderAadapter` instead?